lighthouse-treemap/app/src/main.js

```diff
@@ -287,6 +293,91 @@ class TreemapViewer {
     this.updateColors();
   }
 
+  createTable() {
+    const tableEl = TreemapUtil.find('.lh-table');
+    tableEl.innerHTML = '';
+
+    /** @type {Array<{name: string, bytes: {resource: number, unused?: number}}>} */
+    const data = [];
+    TreemapUtil.walk(this.currentTreemapRoot, (node, path) => {
+      if (node.children) return;
+
+      // Elide the first path component, which is common to all nodes.
+      let name;
+      if (path[0] === this.currentTreemapRoot.name) {
+        name = path.slice(1).join('/');
+      } else {
+        name = path.join('/');
+      }
+
+      // Elide the document URL.
+      if (name.startsWith(this.currentTreemapRoot.name)) {
+        name = name.replace(this.currentTreemapRoot.name, '//');
+      }
+
+      data.push({
+        name,
+        bytes: {resource: node.resourceBytes, unused: node.unusedBytes},
+      });
+    });
+
+    const gridEl = document.createElement('div');
+    tableEl.append(gridEl);
+
+    /**
+     * @param {typeof data[0]['bytes']} a
+     * @param {typeof data[0]['bytes']} b
+     * @return {number}
+     */
+    const bytesSorter = (a, b) => a.resource - b.resource;
+
+    const maxSize = this.currentTreemapRoot.resourceBytes;
+    this.table = new Tabulator(gridEl, {
+      data,
+      height: '100%',
+      layout: 'fitColumns',
+      tooltips: true,
+      addRowPos: 'top',
+      history: true,
+      resizableColumns: true,
+      initialSort: [
+        {column: 'bytes', dir: 'desc'},
+      ],
+      columns: [
+        {title: 'Name', field: 'name'},
+        {title: 'Size / Unused', field: 'bytes', sorter: bytesSorter, formatter: cell => {
```

Comment:
Would it be useful to put unused into a separate column so the table can be sorted by most unused bytes?